pyqtgraph/colors/palette.py

```diff
@@ -1,9 +1,7 @@
-from .Qt import QtGui
+from pyqtgraph.Qt import QtGui
```

Comment:
we use relative imports because one of the supported ways this library "can" be used (although we don't recommend it) is by placing a pyqtgraph directory into a project.  Would you mind switching back to a relative import?

Reply:
whops, saw this on the commit, not on the final result, disregard!